shared/templates/coreos_kernel_option/template.py

```diff
@@ -3,4 +3,6 @@
 
 def preprocess(data, lang):
     data["arg_negate"] = parse_template_boolean_value(data, parameter="arg_negate", default_value=False)
+    if "arg_value" not in data:
+        data["arg_value"] = ""
```

Comment:
```suggestion
```

This is probably not necessary. Jinja will be able to properly evaluate `{{% if ARG_VALUE %}}`

Reply:
I got some error at some point and ended up adding it here.